arm-sql/2014-04-01/swagger/secureConnectionPolicy.json

```diff
@@ -0,0 +1,237 @@
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/DatabaseSecureConnectionPolicy"
+                        },
+                        "description": "The required parameters for createing or updating a secure connection policy."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DatabaseSecureConnectionPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DatabaseSecureConnectionPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "SecureConnection"
+                ],
+                "operationId": "Databases_GetSecureConnectionPolicy",
+                "description": "Returns an Azure SQL Database secure connection policy.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database for which the secure connection policy applies."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DatabaseSecureConnectionPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DatabaseSecureConnectionPolicyProperties": {
+            "properties": {
+                "proxyDnsName": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the Data Security Proxy DNS name."
+                },
+                "proxyPort": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the Data Security Proxy port number."
+                },
+                "securityEnabledAccess": {
+                    "type": "string",
```

Comment:
I updated this to be an enum.

---
In reply to: [80166223](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80166223) [](ancestors = 80166223)